src/Symfony/Bundle/TwigBundle/Resources/views/layout.html.twig

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <html>
+    {% block before_head %}{% endblock %}
     <head>
         <meta charset="{{ _charset }}" />
```

Comment:
I may be mistaken, but I have been taught that there should be no content before the `<meta charset="" />`, as – in the case of missing headers – this is the first indication for the user agent as to which encoding applies.

Reply:
I'm not sure if this is an issue. It could only make a difference in the browser. If you'd curl and see the output in the Terminal it wouldn't do anything.

Reply:
https://www.w3.org/International/questions/qa-html-encoding-declarations.en#quickanswer

IMO it does not impact DX if you move the first exception comment block after the `<meta charset ...>`.

Reply:
Changed it.